arborist/__init__.py

```diff
@@ -195,7 +199,15 @@ def study_page(studiesfolder, study):
                            possible_datatypes=possible_datatypes)
 
 
-@app.route(('/folder/<folderpath:studiesfolder>/s/<study>/clinical/create/'))
+@app.route('/folder/<folderpath:studiesfolder>/set_default/', methods=["GET"])
+def set_default_folder(studiesfolder):
+    response = app.make_response(('', 204))
+    response.set_cookie('default_folder', value=studiesfolder)
+    flash('Default folder set.')
```

Comment:
@jochemb Since you don't refresh the page this flash message is only shown on the next refresh, not just after when clicked.